gammapy/background/tests/test_models.py

```diff
@@ -43,3 +49,100 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+class TestCubeBackgroundModel():
+
+    @remote_data
+    def test_read_bin_table(self):
+
+        # test shape of bg cube when reading a file
+        filename = '../test_datasets/background/bg_cube_model_test.fits'
```

Comment:
Looks OK to me no, no worries if you go > 80 characters in some cases that would be awkward to wrap, like filenames or URLs. My point about insisting on line length limit sometimes is that code should be readable, which it isn't if one has to scroll back and forth to read a very long expression.